src/bosh.js

```diff
@@ -767,7 +767,9 @@ Strophe.Bosh.prototype = {
                 // expanding retry window
                 var backoff = Math.min(Math.floor(Strophe.TIMEOUT * this.wait),
                                        Math.pow(req.sends, 3)) * 1000;
-                setTimeout(sendFunc, backoff);
+                setTimeout(function() {
+                    sendFunc();
+                }, backoff);
```

Comment:
Can you please add a small comment to each of those lines to just explain why it's now written this way. Otherwise someone might come along and "improve" it again by removing the enclosing function.